docs/reference/index-modules/index-sorting.asciidoc

```diff
@@ -0,0 +1,102 @@
+[[index-modules-index-sorting]]
+== Index Sorting
+
+When creating a new index in elasticsearch it is possible to configure how the Segments
+inside each Shard will be sorted. By default Lucene does not apply any sort and uses the
+internal _doc_id_ to do the ordering.
+The `index.sort.*` settings define which fields should be used to sort the documents inside each Segment.
+
+[WARNING]
+`nested` fields uses the original sort of the Segment to work which is why they
```

Comment:
`nested` fields are not compatible with index sorting because they rely on the default _doc_id_ sorting. An error will be thrown if index sorting is activated on an index that contains `nested` fields.